src/main/java/com/azure/cosmos/examples/patch/sync/SamplePatchQuickstart.java

```diff
@@ -0,0 +1,592 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.examples.patch.sync;
+
+import com.azure.cosmos.BulkOperations;
+import com.azure.cosmos.ConsistencyLevel;
+import com.azure.cosmos.CosmosBulkOperationResponse;
+import com.azure.cosmos.CosmosClient;
+import com.azure.cosmos.CosmosClientBuilder;
+import com.azure.cosmos.CosmosContainer;
+import com.azure.cosmos.CosmosDatabase;
+import com.azure.cosmos.CosmosException;
+import com.azure.cosmos.CosmosItemOperation;
+import com.azure.cosmos.CosmosPatchOperations;
+import com.azure.cosmos.TransactionalBatch;
+import com.azure.cosmos.TransactionalBatchOperationResult;
+import com.azure.cosmos.TransactionalBatchPatchItemRequestOptions;
+import com.azure.cosmos.TransactionalBatchResponse;
+import com.azure.cosmos.examples.common.AccountSettings;
+import com.azure.cosmos.examples.common.Families;
+import com.azure.cosmos.examples.common.Family;
+import com.azure.cosmos.examples.common.Child;
+import com.azure.cosmos.models.CosmosContainerProperties;
+import com.azure.cosmos.models.CosmosContainerResponse;
+import com.azure.cosmos.models.CosmosDatabaseResponse;
+import com.azure.cosmos.models.CosmosItemRequestOptions;
+import com.azure.cosmos.models.CosmosItemResponse;
+import com.azure.cosmos.models.CosmosPatchItemRequestOptions;
+import com.azure.cosmos.models.CosmosQueryRequestOptions;
+import com.azure.cosmos.models.PartitionKey;
+import com.azure.cosmos.models.ThroughputProperties;
+import com.azure.cosmos.util.CosmosPagedIterable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class SamplePatchQuickstart {
+
+    private CosmosClient client;
+
+    private final String databaseName = "AzureSampleFamilyDB";
+    private final String containerName = "FamilyContainer";
+
+    private CosmosDatabase database;
+    private CosmosContainer container;
+
+    protected static Logger logger = LoggerFactory.getLogger(SamplePatchQuickstart.class);
+
+    public void close() {
+        client.close();
+    }
+
+    /**
+     * Run a Hello CosmosDB console application.
+     * <p>
+     * This is a simple sample application intended to demonstrate Create, Read,
+     * Update, Delete (CRUD) operations with Azure Cosmos DB Java SDK, as applied to
+     * databases, containers and items. This sample will 1. Create synchronous
+     * client, database and container instances 2. Create several items 3. Upsert
+     * one of the items 4. Perform a query over the items 5. Delete an item 6.
+     * Delete the Cosmos DB database and container resources and close the client. *
+     */
+    // <Main>
+    public static void main(String[] args) {
+        SamplePatchQuickstart p = new SamplePatchQuickstart();
```

Comment:
Don't create variables such as "i", "j" ... Lets give these some meaningful names specially when we are thinking about giving users.

Reply:
Agreed :) FYI - The setup, tear-down etc. have been picked up from other sample (see `crudquickstart\sync` folder) and have been left untouched  to maintain consistency. Only the Patch related methods have been added